### PR TITLE
PACTA - New config file for project initialisation

### PR DESCRIPTION
**Pull request by jacobvjk**
This Rmd describes the changes to the workflow in the PACTA_analysis repo due to a new config file for project initialisation

### CODE REVIEW

**Comment by jacobvjk** [comment]
@maurolepore this is the draft for the blog post. I changed the text a bit to better match the changes to the repo's README

File: config_project_settings [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: ''
```
Comment (maurolepore) [inline config_project_settings]: What title you suggest?
Also, can you suggest a description (1 sentence or phrase)

File: config_project_settings [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+author: "Jacob Kastl"
```
Comment (maurolepore) [inline config_project_settings]: Do you have a personal website? Maybe your GitHub profile.

File: config_project_settings (hunk 1 above)
Comment (maurolepore) [reply config_project_settings]: Oh, sorry, I now see this below.

File: config_project_settings (hunk 2 above)
Comment (maurolepore) [reply config_project_settings]: Sorry, I now see this too.

File: config_project_settings [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+output: 
+  distill::distill_article:
+    toc: true
+    toc_depth: 3
+    self_contained: true
+categories:
+  - other
+  - programming
+twitter:
+  site: "@JacKastl"
```
Comment (maurolepore) [inline config_project_settings]: :heart:

File: config_project_settings [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+**Target Audience:**
```
Comment (maurolepore) [inline config_project_settings]: Consider replacing `**Heading**` with `## Heading`. That will make your headings show up in the table of contents, which then becomes a nice overview of the article.

**Review by maurolepore — APPROVED** [review]
I moved your article to _posts/ and did minor changes to formatting. I later saw some changes you pushed recently and I may have missed those. Please review this file and do any edits you think necessary here.

**Review by jacobvjk — COMMENTED** [review]
This looks good, nothing got messed up

**Pull request merged**